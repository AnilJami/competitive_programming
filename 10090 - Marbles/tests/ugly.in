9
1
2
4
7
10
16
75
85
150