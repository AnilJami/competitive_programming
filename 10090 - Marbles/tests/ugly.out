The No.1 ugly number is 1.
The No.2 ugly number is 2.
The No.4 ugly number is 4.
The No.7 ugly number is 8.
The No.10 ugly number is 12.
The No.16 ugly number is 25.
The No.75 ugly number is 675.
The No.85 ugly number is 972.
The No.150 ugly number is 5832.
